variables.tf

```diff
@@ -77,6 +77,23 @@ variable "create_cos_bucket" {
   default     = true
 }
 
+variable "cross_region_location_enabled" {
+  description = "Retention enabled for COS bucket. Only used if 'create_cos_bucket' is true."
+  type        = bool
+  default     = false
+}
+
+variable "cross_region_location" {
+  description = "Specify the cross-regional bucket location. Supported values are 'us', 'eu', and 'ap'. If you pass a value for this, do not pass one for var.region."
```

Comment:
Actually, might be clearer if said "If you pass a value for this, ensure to set the value of var.region to null"